r/src/compute.cpp

```diff
@@ -449,7 +449,7 @@ std::shared_ptr<arrow::compute::FunctionOptions> make_compute_options(
     return std::make_shared<Options>(cpp11::as_cpp<std::string>(options["characters"]));
   }
 
-  if (func_name == "utf8_slice_codeunits") {
+  if (func_name == "utf8_slice_codeunits" || func_name == "binary_slice_bytes") {
```

Comment:
Should a test be created on the R side?
@thisisnic @paleolimbot Perhaps one of you can help.